eclipse/src/saros/ui/widgets/wizard/EnterXMPPAccountComposite.java

```diff
@@ -90,6 +97,20 @@ public void expansionStateChanged(ExpansionEvent e) {
           }
         });
 
+    createServerExpandableCompositeContent();
+
+    if (CreateXMPPAccountWizard.CREATE_DIALOG_ENABLED) {
```

Comment:
I do not understand the whole checks. I mean you cannot click on a button that is either disabled or not visible. So there should be only one check in the whole class or none at all.

Reply:
Ahhh sure. Thank you :+1:
I just need
```java
createAccountButton.setVisible(CreateXMPPAccountWizard.CREATE_DIALOG_ENABLED)
```